unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -509,6 +509,16 @@ private void HideAll()
             }
         }
 
+        public void ShowAllRenderers()
+        {
+            Renderer[] renderers = gameObject.GetComponentsInChildren<Renderer>();
+
+            for (int i = 0; i < renderers.Length; i++)
+            {
+                renderers[i].enabled = true;
+            }
+        }
+
```

Comment:
every `WearableController` already has a list of all the renderers involved, so we could traverse the `wearableControllers` collection and call their `SetAssetRenderersEnabled(false/true)` to toggle the renderers.

If that works, we avoid calling GetComponentsInChildren which is costly. If it doesn't work out of the box like that, you can disregard this comment